packages/react-ui-core/src/Ratings/RatingItem.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+export default class RatingItem extends Component {
+  static propTypes= {
+    theme: PropTypes.object,
+  }
+  static defaultProps = {
+    theme: {},
+  }
+
+  render() {
+    const {
+      width,
+      index,
+      backgroundColor,
+      theme,
+    } = this.props
+    const id = `rItem_${index}`
+    return (
+      <div className={theme.ratingitem}>
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="50"
+        height="50"
+        viewBox="0 0 51 48"
+      >
+        <defs>
+          <linearGradient id={id}>
+            <stop offset="0%" stopOpacity="1" stopColor={backgroundColor}/>
+            <stop offset={width} stopOpacity="1" stopColor={backgroundColor} />
+            <stop offset={width} stopOpacity="0" stopColor={backgroundColor} />
+            <stop offset="100%" stopOpacity="0" stopColor={backgroundColor} />
+          </linearGradient>
+        </defs>
+        <title>Five Pointed Star</title>
+        <path
+          fill={`url(#${id})`}
```

Comment:
color fill is done using offset property inside the linear gradient tag.
the fill over here should contain unique id that matches that of linear gradient because, I'm iterating an array of elements and each should have an unique id else all elements in the array are assuming the properties of first array element, which is undesirable.